drivers/gles3/storage/material_storage.cpp

```
@@ -3047,6 +3052,10 @@ void SceneShaderData::set_code(const String &p_code) {
 	if (uses_normal_texture) {
 		WARN_PRINT_ONCE_ED("Reading from the normal-roughness texture is only available when using the Forward+ or Mobile renderers.");
 	}
+
+	if (specular_mode == BaseMaterial3D::SPECULAR_MULTISCATTERING_GGX) {
+		WARN_PRINT_ONCE_ED("MultiscatterGGX is only available when using the Forward+ renderer.");
```

I am not certain how much sense it makes to print this warning. It is a well known fact that the compat renderer is not designed for the utmost visual quality, so imo it should just silently fall back to Schlick. There is no reason to warn the user about something they can't change.

---

I implemented it this way because I noticed a warning is triggered for each feature that doesn't work on Mobile and Compatibility. Even if we decide to keep multiscattering as default, this warning would still be relevant since multiscattering won't work on Mobile and Compatibility due to the lack of a DFG LUT - which I'm in favor of not adding.